src/components/testLevelSelector/testLevelSelectorItem/testLevelSelectorItem.jsx

```diff
@@ -9,9 +10,11 @@ const TestLevelsSelectorItem = ({ title, description }) => {
     <div className='test-level-selector-item'>
       <div className="title">{title}</div>
       <div className="description">{description}</div>
-      <Button variant="contained" color="primary" className='btn'>
+      <Link to="/test" className='btn'>
```

Comment:
You can write link button like this:
https://github.com/mtalstykh-exadel/team4_frontend/blob/a146ae45a8b03735d6d53da1b924d024244cc9fd/src/components/header/header.js#L70-L76